### PR TITLE
(CLOUD-196) Expose instance security groups to puppet resource

### PR DESCRIPTION
**Pull request by garethr**

### CODE REVIEW

File: spec/unit/type/ec2_instance_spec.rb [hunk 1]
```diff
@@ -7,13 +7,14 @@
...
       :name,
-      :security_groups
     ]
   end
 
   let :properties do
     [
       :ensure,
+      :security_groups,
+      :tags,
```
Comment (cmurphy) [inline spec/unit/type/ec2_instance_spec.rb]: Is this relevant to this PR?

Comment (garethr) [inline spec/unit/type/ec2_instance_spec.rb]: Note the tags line. I can push that into master directly if you'd prefer.

Comment (Iristyle) [inline spec/unit/type/ec2_instance_spec.rb]: We scrapped `tags` from this commit... so feel free to put it in #53 or add a `(maint)` commit to master if you'd like.
